packages/fuels-core/src/json_abi.rs

```diff
@@ -237,6 +237,7 @@ impl ABIParser {
 
                 Ok(Token::Enum(Box::new((discriminant as u8, token))))
             }
+            _ => Err(Error::InvalidType(param.to_string())),
```

Comment:
I think there should be `Token::Tuple` somewhere?

Reply:
This (`tokenize`) is actually only used for the JSON ABI CLI tool, which is out of the scope for this issue/PR. The SDK works without any of this `tokenize()`. I've updated it to include a `todo!()` in here instead of `Error`, and the issue for future work is here https://github.com/FuelLabs/fuels-rs/issues/206.

Reply:
Oh ok! Just realized the role of json_abi wasn't clear for me then haha